netdata-installer.sh

```diff
@@ -534,6 +534,7 @@ if [ ! -f "${NETDATA_PREFIX}/etc/netdata/.installer-cleanup-of-stock-configs-don
 			t="${f/.conf.installer_backup.*/.conf}"
 			t="${t/.conf.old/.conf}"
 			t="${t/.conf.orig/.conf}"
+			t="${t/orig\//}"
```

Comment:
In this `if` section file cannot be directory, so it cannot end with `/` thus this is not needed.

Reply:
> In this `if` section file cannot be directory, so it cannot end with `/` thus this is not needed.

Allow me to explain with an example and let me know what i miss
This is what we trying to achieve, we have an original path as seen from the find:
```
root@pve:~/netdata# x="/etc/netdata/orig/python.d/logind.conf.orig" 
```

And we transform it like this:
```
root@pve:~/netdata# f="${x/${NETDATA_PREFIX}\/etc\/netdata\//}"
root@pve:~/netdata# # find the stock filename
root@pve:~/netdata# t="${f/.conf.installer_backup.*/.conf}"
root@pve:~/netdata# t="${t/.conf.old/.conf}"
root@pve:~/netdata# t="${t/.conf.orig/.conf}"
root@pve:~/netdata# t="${t/orig\//}"
root@pve:~/netdata# 
root@pve:~/netdata# echo $t
python.d/logind.conf
root@pve:~/netdata# 
```

So that we can then match the signatures:
```
root@pve:~/netdata# grep 'python.d/logind.conf' configs.signatures 
  ['46dfa2b6a7e7c76532e00c1344d5d171']='python.d/logind.conf'
root@pve:~/netdata# 
```

Can you clarify a bit further what you meant?

Reply:
ok, in this case it is ok.